build/sphinx/homepage-config/conf.py

```diff
@@ -1,13 +1,14 @@
 # -*- coding: utf-8 -*-
-from docutils import nodes
 import os
 import sys
 
+from docutils import nodes
+
 # Import common docs config.
 this_path = os.path.dirname(__file__)
 sys.path.append(os.path.normpath(os.path.join(this_path, '../')))
 
-from mongoc_common import *
+from mongoc_common import *  # noqa: E402, F403
```

Comment:
Could this be rewritten to instead import the variables that are actually relevant for the sphinx config, rather than star-importing everything?